doc/programming_guide/asrc/overview.rst

```diff
@@ -2,11 +2,32 @@
 Overview
 ********
 
+.. warning::
+
+   This example is based on the RTOS framework and drivers. This choice allowed simplifying the example design, but it lead to some latency in the system.
+   The main sources of latency are:
+
+      - Large block size used for ASRC processing: this is necessary to minimise latency associated with the intertile context and thread switching overhead.
+      - Large size of the buffer to which the ASRC output samples are written: a stable level (half full) must be reached before starting streaming out over USB.
+      - RTOS task scheduling overhead between the tasks.
+      - bInterval of USB in the RTOS drivers is set to 4, i.e. one frame every 1ms.
+      - Block based implementation of the USB and I2S RTOS drivers.
```

Comment:
`I2S` -> `|I2S|`

Reply:
Fixed in https://github.com/xmos/sln_voice/pull/365/commits/b2580e8b8ca150967fe31bfc96e2d2e55f7c46a3.